Adding text for color accessibility (#283)
primer/design: content/foundations/color.mdx

  src="https://user-images.githubusercontent.com/378023/156498652-b1121c86-a670-4a92-9a8d-9db3c7e0ed9a.png"
/>

## Accessibility
When designing with colors consider the following to make your UI visually accessible.

### Assure adequate color contrast

> Color contrast between text and its background must meet [required WCAG standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). 

Reviewer: As the [functional role pairing](https://primer.style/design/foundations/color#combining-colors) guarantees WCAG compliance already, should we put part of this content within the "Combining colors" section? For instance, describing what an accessible combination means and reinforcing the usage of the functional pairing. Maybe the images with combinations can be improved by adding the contrast ratio to go hand in hand with the content.

Reviewer: 👍 

To the mention below, I'd use "functional colors" or "functional role pairing" instead of "primer colors"

Author: Good one, changed that. Thanks.

> When you combine colors like foreground and background, they must fulfil the [required wcag contrast](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). 

Reviewer 2: ```suggestion
Color contrast between text and its background must meet [required WCAG standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). 
```

  src="https://user-images.githubusercontent.com/378023/156498652-b1121c86-a670-4a92-9a8d-9db3c7e0ed9a.png"
/>

## Accessibility
When designing with colors consider the following to make your UI visually accessible.

### Assure adequate color contrast
Color contrast between text and its background must meet [required WCAG standards](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html). 

The contrast requirements are:
- 4.5:1 for normal text
- 3:1 for large text (>24px)
- 3:1 for UI elements and graphics

> - Decorative and disabled elements have no contrast requirement

Reviewer: nitpick (non-blocking): This last item seems a bit off. Should we put this as part of the introduction or as a legend?

e.g.: `The contrast requirements for non-decorative and active elements are:`

Author: Why do you feel this is off? I would think (and ahd this myself) that when you read about contrast requirements it would make sense to include the requirements for disabled and decorative items. 

We could turn it around to fit more into the style:
`No contrast requirement for decorative and disabled elements`

Reviewer: The three first items in the list are requirements to meet the criteria, while the last point is not a requirement, but an observation. 

> We could turn it around to fit more into the style:
> `No contrast requirement for decorative and disabled elements`
> 

That sounds a bit better 👍

Author: Or you could say its a "non-requirement". As it is specifically listed in the specs.

Will update the sentence order for now.

Use an [online contrast checker](https://webaim.org/resources/contrastchecker/) or a [figma plugin](https://primer.style/design/accessibility/tools#contrast-plugin) to test your contrast. 

> If you work with the [primer colors](#color-roles-recipes), this work has already been done for you. We checked all recommended combinations, so you are all set.

Reviewer: ```suggestion
If you work with the [functional color combinations](#color-roles-recipes), this work has already been done for you. We checked all recommended combinations, so you are all set.
```

> ### Don't rely on color alone

Reviewer 3: 💯

<Box mb={3} display="flex" alignItems="flex-start" alignItems="center" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
  <Box as="p" mt="0">
    <Heading sx={{mt: 0, mb: 3, fontSize: 2}} as="h4">Show state with more than color</Heading>
    Color vision deficiency is different for different people. To make sure everyone can understand and use your UI you should show state with more than a change in color.
For example by using icons or changing the content.
  </Box>
  <img
    width="456"

>     alt="Icon supporting color to show error"

Author: ❤️

    src="https://user-images.githubusercontent.com/813754/187186864-8a825434-978b-4e21-bac2-9c1235d75e2c.png"
  />
</Box>

<Box mb={3} display="flex" alignItems="flex-start" alignItems="start" flexDirection={["column", "column", "column", "column", "row"]} sx={{gap: 3}}>
  <Box as="p" mt="0">
      <Heading sx={{mt: 0, mb: 3, fontSize: 2}} as="h4">Connect labels to graphs with lines or patterns</Heading>
      For charts and graphs you can position the labels on top or close to each section.
      You can also use patterns to distinguish different parts.
  </Box>
  <img
      width="456"

>       alt="Graph with accessible legend"

Reviewer 3: ```suggestion
      alt="Two graphs. One is a line graph and one is a pie graph. The line graph is demonstrating using unique shapes for each comparison line's plot symbol. The pie chart is demonstrating how each slice's label can use a line to point to its corresponding slide."
```

Adding a little more detail here about how different distinguishing techniques can be utilized.
